### PR TITLE
Implement audio classification widget

### PR DESCRIPTION
**Pull request by mishig25**
Implemented audio classification widget.

Request to Inference API (same as input for ASR, Audio-to-Audio widgets):
```js
const requestBody = { file }; // file is an audio file
```

Response from Inference API (same as Image Classification Widget):
```
Array<{ label: string; score: number }>
```

Find this [slack thread](https://huggingface.slack.com/archives/C02D0F0TGRW/p1630507899016300) for more 

FYI, @patrickvonplaten @anton-l 

Screenshots:
| initial state                                                                                                                                                         | success state                                                                                                                                                         | fail state                                                                                                                                                            |
|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Screenshot 2021-09-03 at 11 52 29 | Screenshot 2021-09-03 at 11 54 35 | Screenshot 2021-09-03 at 11 54 20 |

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Awesome - should we merge this? :-)

**Comment by mishig25** [comment]
@patrickvonplaten already pinged @beurkinger :) should be soon

File: widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	<svelte:fragment slot="before">
 		<input
 			bind:group
-			class="bg-gray-200 border-transparent focus:border-transparent text-blue-500 focus:ring-1 focus:ring-offset-2 focus:ring-gray-200 mr-2"
+			class="bg-gray-200 dark:bg-gray-700 shadow-inner border-transparent focus:border-transparent text-blue-500 ring-1 ring-offset-2 ring-gray-200 dark:ring-gray-700 mr-2"
```
Comment (beurkinger) [inline widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte:15]: What's the reason behind this change ?

**Comment by beurkinger** [comment]
Please include some screenshots :)

File: widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte (hunk 1 above)
Comment (mishig25) [reply widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte:15]: Two reasons:
1. The radio was looking very flat and unrecognizable. By enabling the ring and adding inner-shadow it definitely looks like a radio
2. Dark mode support

**Comment by mishig25** [comment]
Added them :)

File: widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte (hunk 1 above)
Comment (beurkinger) [reply widgets/src/lib/InferenceWidget/shared/WidgetRadio/WidgetRadio.svelte:15]: Alright ! I now see it in the screenshots. Tagging @gary149 so he can have a look if he wants to

**Comment by osanseviero** [comment]
FYI @beurkinger these PRs automatically deploy netlify with the changes so you can try the widgets directly https://6131e20a431ea80d15e1809d--huggingface-widgets.netlify.app/

**Review by julien-c — APPROVED** [review]
looks great!
